edgelet/contrib/debian/preinst

```diff
@@ -12,6 +12,11 @@ add_groups()
     fi
     mkdir -p /var/lib/aziot/edged
 
+    # Create an edgeagentuser and add it to iotedge group
+    if ! getent passwd edgeagentuser >/dev/null; then
+        useradd -g iotedge -c "edgeAgent user" -ms /bin/bash -u 13622 edgeagentuser
```

Comment:
Why is this needed here? How do we make sure the UID here is same as in the EA docker file?

Reply:
Dockerfile is present for backward compatibility only. It will go away eventually.
The uuid in docker file is also harcoded.

Our user can change ithat 13622 in dockerfile but it is ok. the mgmt ownership is also re-assigned in dockerfile so it still works.